scripts/lib/converter.go

```diff
@@ -18,6 +19,9 @@ type TextConverter struct {
 	// value: display name
 	users map[string]string
 	re    regexps
+	// 公開するサイトのURLのベース
+	// 環境変数BASEURLで指定する
+	baseURL string
```

Comment:
goでフィールドへのコメントを付けるときはフィールド名で始めるようにする

```suggestion

	// baseURL: 公開するサイトのURLのベース。
	// 環境変数BASEURLで指定する
	baseURL string
```

あと1行空けてしまったほうが良い

なお英語のほうがより自然で以下のような感じになる

```go
// baseURL is root path for public site, configured by `BASEURL` environment variable.
```

Reply:
英語のコメントの方、使わせて頂きました